### PR TITLE
Add Norwegian translation

### PR DESCRIPTION
**Pull request by perandre**

### CODE REVIEW

File: app/locales/no.ts [hunk 1]
```diff
@@ -0,0 +1,186 @@
+import { SubmitKey } from "../store/config";
+import type { LocaleType } from "./index";
+
+const en: LocaleType = {
```
Comment (xavierbarbosa) [inline app/locales/no.ts:4]: Should be:
```tsx
const no: LocaleType = {
   // ... translations here
};

export default no;
```

**Comment by perandre** [comment]
@xavierbarbosa I tried to add vi lang to the mix, can you check why it's failing?

**Comment by xavierbarbosa** [comment]
@perandre `vi.ts` file is completely missing. You forgot to add it?

**Comment by perandre** [comment]
> @perandre `vi.ts` file is completely missing. You forgot to add it? I would maybe make this PR only about this language and add Vietnamese on a different PR.

The missing file shouldn't cause a conflict ... it lists all the other files as conflicting.

**Comment by binh234** [comment]
Hi @perandre, I added the Vietnamese translation some days ago. I think you will need to resolve the conflicts and merge with the upstream repository to keep your work up-to-date.

**Comment by perandre** [comment]
I'll sync again and try later; some issues with GH today it seems.

**Comment by Yidadaa** [comment]
Hello everyone, is there any update on this PR? If the conflicts have been resolved, please @ me and I will promptly approve it.

**Comment by xavierbarbosa** [comment]
@Yidadaa Fixed the conflicts.

**Pull request merged**